lib/FusionInventory/Agent/SNMP/Live.pm

```diff
@@ -64,7 +64,7 @@ sub new {
             -varbindlist => [$oid]
         );
         die "no response from host $params{hostname}\n"
-            if !$response;
+            if !$response || !$response->{$oid};
```

Comment:
Doesn't the second condition implies a a response different from the one expected, rather than a lack of response ? In this case, I'd rather issue a different error message.